### PR TITLE
Added blinds rotation for the motors that support it

### PR DESCRIPTION
**Pull request by marekhalmo**

### CODE REVIEW

**Comment by marekhalmo** [comment]
Releated issue #33

File: src/nodes/tahoma.ts [hunk 1]
```diff
@@ -93,6 +93,13 @@ export = (RED: Red) => {
                     statusDoneText = 'Set to ' + msg.payload.position + '%';
                     expectedState = { open: true, position: msg.payload.position };
                     break;
+                case 'customRotation':
+                    commandName = 'rotation';
+                    parameters = [{ name: 'orientation', value: parseInt(msg.payload.orientation, 10) }];
+                    statusProgressText = 'Rotating to ' + msg.payload.orientation + 'degrees...';
+                    statusDoneText = 'Rotated to ' + msg.payload.orientation + '%';
```
Comment (nikkow) [inline src/nodes/tahoma.ts:100]: Hi @marekhalmo, thanks for the PR 😃!
Lines 99 and 100 use both a different "unit" (one is `degrees` and the other `%`). I just want to make sure this is intentional :)

**Comment by marekhalmo** [comment]
Ok changed to degree symbol

**Comment by marekhalmo** [comment]
added text to the md file as well...

**Pull request merged**